### ISSUE TITLE
[Feature Request] Show temperature in menubar side by side with battery percent.

### ISSUE THREAD

**Issue by Eric-Guo** [davidwernhart/AlDente#489]
Would be great if can display the temperature at the same time of battery percent, like up/down style.

Before (network speed is another app, not relative with AIDente): 

<img width="165" alt="origin" src="https://user-images.githubusercontent.com/1131536/144694642-8e2b26e3-7057-40db-9b88-fdbfec5d5116.png">

After
![after](https://user-images.githubusercontent.com/1131536/144694649-916d4b03-888c-4d05-b745-d1bced45b316.jpg)
:

**Comment by Stickyhavr** [comment]
Yes. I second the request. I came here to start a new feature request but saw that my requests already exist in this post (#489) and also #620. :-)

**Comment by basmets6** [comment]
+1 for me as well. Would be super useful if you actively use the "Heat Protection" feature.

I purchased the full version, and a couple of times my MBP just wasn't charging up to the limit.
A bit stumped, I figured out that it was due to the Heat Protection.. Then I opened the settings menu and waited, waited, waited until temp dropped.

Would therefore be super useful to have this in the menu bar, so you don't have to keep that window open the entire time.

**Comment by MatthiasKerbl** [comment]
Hi everyone,

With AlDente 1.36, we have introduced fully customizable menubar items. You can now choose stats you want to see in the menubar, such as the battery temperature.

Best,
Matthias
